AppCenter/AppCenter.xcodeproj/project.pbxproj

```diff
@@ -1475,6 +1475,10 @@
 		3844FF161E8C261D003E9194 /* Model */ = {
 			isa = PBXGroup;
 			children = (
+				3592ABA61DC90E3600EF4592 /* MSLogger.m */,
```

Comment:
This shouldn't be here. I meant `MSLog`, not `MSLogger`. You should be able to find `MSLog.h` file that isn't under `Model` folder. Please look at the file in Finder.

Reply:
Reset the `MSLogger` location.